CHANGELOG/CHANGELOG-1.10.md

```diff
@@ -0,0 +1,288 @@
+# v1.10.0 - Changelog since v1.9.2
+
+## Changes by Kind
+
+### Feature
+- It is no longer necessary to specify allowedTopologies for zonal -> zonal cloning and zonal/regional -> regional cloning to ensure that the clone zone is compatible with the GCE volume cloning requirements. ([#1150](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1150), [@amacaskill](https://github.com/amacaskill))
+
+### Bug or Regression
+
+- Fix missing libedit.so.2 error ([#1177](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1177), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
+- Set concurrency limit to prevent OOMing when issuing multiple concurrent mkfs calls. ([#1169](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1169/commits/7f1e04e15631b14f7d3f441cc97bed1a968f2e47), [@artemvmin](https://github.com/artemvmin))
+
+### Other (Cleanup or Flake)
+
+- Update go version to 1.20.3 for k/k 1.27 ([#1180](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1180), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
+- Upgrade mount-utils to v0.27.0. ([#1169](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1169/commits/232bd0aa691a72c1bd806d8c4ed881bb3a47174f), [@artemvmin](https://github.com/artemvmin))
+- Break dependency on k/k test/e2e/storage/podlogs. ([#1169](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1169/commits/1b6e7e537bbc264a1c076c317e6194a976509011), [@artemvmin](https://github.com/artemvmin))
```

Comment:
@artemvmin Is this an accurate release note for the break  k/k dependency commit?

Reply:
Correct, that was the only package using k/k.